### PR TITLE
Eject and setup env for GA

### PR DESCRIPTION
**Pull request by namdaoduy**

### CODE REVIEW

**Review by vudung45 — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -71,6 +73,7 @@ jobs:
             -e CLIENT_ID="$CLIENT_ID" \
             -e CLIENT_SECRET="$CLIENT_SECRET" \
             -e REDIS_URL="$REDIS_URL" \
+            -e REACT_APP_ENV="$REACT_APP_ENV" \
```
Comment (vudung45) [inline .github/workflows/deploy.yml:76]:
```suggestion
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -83,6 +86,7 @@ jobs:
           CLIENT_ID: ${{ secrets.CLIENT_ID }}
           CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
           REDIS_URL: ${{ secrets.REDIS_URL_STAGING }}
+          REACT_APP_ENV: dev
```
Comment (vudung45) [inline .github/workflows/deploy.yml:89]:
```suggestion

```

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -106,4 +111,5 @@ jobs:
             -e CLIENT_ID="$CLIENT_ID" \
             -e CLIENT_SECRET="$CLIENT_SECRET" \
             -e REDIS_URL="$REDIS_URL" \
+            -e REACT_APP_ENV="$REACT_APP_ENV" \
```
Comment (vudung45) [inline .github/workflows/deploy.yml:114]:
```suggestion
            -e REACT_APP_ENV=dev \
```

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -48,6 +48,7 @@ jobs:
           CLIENT_ID: ${{ secrets.CLIENT_ID }}
           CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
           REDIS_URL: ${{ secrets.REDIS_URL_STAGING }}
+          REACT_APP_ENV: dev
```
Comment (vudung45) [inline .github/workflows/deploy.yml:51]:
```suggestion
```

File: .github/workflows/deploy.yml [hunk 5]
```diff
@@ -60,7 +61,8 @@ jobs:
           LMS_BASE_URL,
           CLIENT_ID,
           CLIENT_SECRET,
-          REDIS_URL
+          REDIS_URL,
+          REACT_APP_ENV
```
Comment (vudung45) [inline .github/workflows/deploy.yml:65]:
```suggestion
          REDIS_URL
```

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (vudung45) [inline .github/workflows/deploy.yml:76]:
```suggestion
            -e REACT_APP_ENV=dev \
```

File: .github/workflows/deploy.yml [hunk 6]
```diff
@@ -95,7 +99,8 @@ jobs:
...
```
Comment (vudung45) [inline .github/workflows/deploy.yml:103]:
```suggestion
          REDIS_URL
```

**Review by vudung45 — APPROVED** [review]

**Comment by manhhung741** [comment]
Da fix cai Redis env chua the?

**Pull request merged**